conf/messages

```diff
@@ -21,3 +21,10 @@ cf.not-subscribed-to-cds.detail.eori-info-p1=You can
 cf.not-subscribed-to-cds.detail.eori-link=apply for an EORI number
 cf.not-subscribed-to-cds.detail.eori-info-p2=and return to the Government Gateway login page once your EORI number is associated with your login details.
 
+
+# Supporting Evidence: Holding Page
+supporting-evidence.holdingPage.title = Document check
+supporting-evidence.holdingPage.heading = Wait a few seconds then select 'continue'.
```

Comment:
use this to put the proper quotes in: https://smartquotesforsmartpeople.com/

the ones you have are not correct and don't render as we need them to